core/Controller/ProfileApiController.php

```diff
@@ -53,6 +54,14 @@ public function __construct(
 	 * @NoSubAdminRequired
 	 * @PasswordConfirmationRequired
 	 * @UserRateThrottle(limit=40, period=600)
+	 * @param string $targetUserId
+	 * @param string $paramId
+	 * @param string $visibility
+	 * @return DataResponse
+	 * @throws OCSBadRequestException
+	 * @throws OCSForbiddenException
+	 * @throws OCSNotFoundException
+	 * @throws Exception
```

Comment:
```suggestion
	 * @throws OCSBadRequestException
	 * @throws OCSForbiddenException
	 * @throws OCSNotFoundException
```

The other things should not be needed as they are indicated by the method signature or not planned to happen/be catched (general exception)

Reply:
another open comment here